ArjunaCore/arjuna/classes/com/arjuna/ats/arjuna/coordinator/ActionManager.java

```diff
@@ -86,12 +87,17 @@ public int getNumberOfInflightTransactions()
     {
         return _allActions.size();
     }
+		
+		public boolean areThereInFlightTransactions()
+		{
```

Comment:
I wonder if we couldn't use `getNumberOfInflightTransactions() == 0` instead?

Reply:
Yep, no problem

Reply:
Done